sdk/monitor/Azure.Monitor.OpenTelemetry/tests/Examples.AspNetCore/appsettings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "AzureMonitorOpenTelemetry": {
+        "ConnectionString": "InstrumentationKey=00000000-0000-0000-0000-000000000000",
+        "EnableTraces": true,
+        "EnableMetrics": true,
+        "EnableLogs": true,
+        "StorageDirectory": "",
+        "DisableOfflineStorage": false
```

Comment:
I guess we want this to be "Enable", too?

Reply:
It is inherited from `AzureMonitorExporterOptions`. We need to change the property in Azure Monitor Exporter.